monitoring.md

```diff
@@ -0,0 +1,206 @@
+# Monitoring `amppackager` in production
+
+Once you've run `amppackager` server in production, you may want to monitor its
+health and performance, as well as the performance of the underlying requests to
+the AMP document server. 
+
+The sections below talk about the things you can monitor, and how to do it.
+
+All the command line examples are for Linux OS.
+
+## Monitoring health
+
+To check `amppackager`'s health, `curl` its `/healthz` endpoint:
+
+```
```

Comment:
Might try appending `shellsession` to this line (or one of its [aliases](https://github.com/github/linguist/blob/2a6d952df870807034c3830738f95ad33222aca0/lib/linguist/languages.yml#L5112-L5113)) to get [syntax highlighting](https://help.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks#syntax-highlighting). (Ditto the rest.)

Reply:
Done.